google-cloud-spanner-cassandra/src/test/resources/valid-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Global client configuration
+globalClientConfigs:
+  spannerEndpoint: "spanner.googleapis.com:443"
+  enableBuiltInMetrics: true
+  healthCheckEndpoint: "127.0.0.1:8080"
+
+# List of all listeners
+listeners:
+  - # Configuration for listener_1
+    name: "listener_1"
+    
+    host: "127.0.0.1"
+    port: 9042
+    enableBuiltInMetrics: false
```

Comment:
do we need `enableBuiltInMetrics` per listener? I think we can start with global setting for now and if/when needed add listener level option